resources/eventing/profile-production.yaml

```diff
@@ -14,15 +14,15 @@ controller:
       cpu: 500m
       memory: 1Gi
     requests:
-      cpu: 400m
+      cpu: 10m
       memory: 512Mi
   publisherProxy:
     resources:
       limits:
         cpu: 500m
         memory: 512Mi
       requests:
-        cpu: 400m
+        cpu: 10m
```

Comment:
Same comment as [this](https://github.com/kyma-project/kyma/pull/14500/files?file-filters%5B%5D=.yaml&owned-by%5B%5D=mfaizanse&show-viewed-files=true#r896752619)